src/main/java/io/meeds/qa/ui/pages/page/factory/space/SpaceHomePage.java

```diff
@@ -983,6 +987,10 @@ public void checkActivityPinned(String activity) {
     assertWebElementVisible(getPinnedActivity(activity));
   }
 
+  public void pinnedActivityDisappears(String activity) {
+    getPinnedActivity(activity).isNotVisibleAfterWaiting();
```

Comment:
IMO we should use the assert method here instead of isNotVisibleAfterWaiting 
```suggestion
    assertWebElementNotVisible(getPinnedActivity(activity));
```